docker/1.4.12/README.md

```diff
@@ -0,0 +1,49 @@
+#Eclipse Mosquitto v1.4.12 Docker Image
+
+##Mount Points
+
+Three mount points have been created in the image to be used for configuration, persistent storage and logs.
+```
+/mosquitto/config
+/mosquitto/data
+/mosquitto/log
+```
+
+
+##Configuration
+
+When running the image, the default configuration values are used.
+To use a custom configuration file, mount a **local** configuration file to `/mosquitto/config/mosquitto.conf`
+```
+docker run -it -p 1883:1883 -p 9001:9001 -v <path-to-configuration-file>:/mosquitto/config/mosquitto.conf mosquitto:1.4.10
+```
+
+Configuration can be changed to:
+
+* persist data to `/mosquitto/data`
+* log to `/mosquitto/log/mosquitto.log`
+
+i.e. add the following to `mosquitto.conf`:
+```
+persistence true
+persistence_location /mosquitto/data/
+
+log_dest file /mosquitto/log/mosquitto.log
+```
+
+**Note**: If a volume is used, the data will persist between containers.
+
+##Build
+Build the image:
+```
+docker build -t mosquitto:1.4.10 .
+```
+
+##Run
+Run a container using the new image:
+```
+docker run -it -p 1883:1883 -p 9001:9001 -v <path-to-configuration-file>:/mosquitto/config/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.10
```

Comment:
I was in a hurry and completely missed those. Should be fixed now.

Also I've updated docker README to use the official image name from docker hub.